src/index.js

```diff
@@ -44,7 +44,7 @@ export function rangeFromLineNumber(textEditor: TextEditor, line: ?number, colum
     }
   }
   if (colStart > lineText.length) {
-    throw new Error(`Column start (${colStart}) greater than line length (${lineText.length})`)
+    throw new Error(`Column start (${colStart || 0}) greater than line length (${lineText.length})`)
```

Comment:
This isn't necessary, we have already verified that the given column is a number, or set to `0` on 31-32.